x11-drivers/xf86-video-virtualbox/xf86-video-virtualbox-5.1.22.ebuild

```diff
@@ -58,6 +58,9 @@ PATCHES=(
 
 	# xorg-1.19 patch from opensuse (bug #602784)
 	"${FILESDIR}/${PN}-5.1.10-xorg119.patch"
+
+	# fix bug #579946
+	"${FILESDIR}/${PN}-5.1.22-sysmacros.patch"
```

Comment:
No, it's in `x11-drivers/xf86-video-virtualbox/files/xf86-video-virtualbox-5.1.22-sysmacros.patch` file